lib/main_common.pm

```diff
@@ -2401,116 +2401,186 @@ sub load_virtualization_tests {
 }
 
 sub load_hypervisor_tests {
+    # Check if the HOST_HYPERVISOR variable is set to 'xen', 'kvm', or 'qemu'
     return unless (get_var('HOST_HYPERVISOR') =~ /xen|kvm|qemu/);
-    return unless get_var('VIRT_PART');
-    my $windows = check_var('VIRT_PART', 'windows');
 
-    # Install hypervisor via autoyast or manually
-    loadtest "autoyast/prepare_profile" if get_var "AUTOYAST_PREPARE_PROFILE";
-    load_boot_tests if check_var('VIRT_PART', 'install');
+    # Only proceed if the VIRT_TEST variable is set
+    return unless (get_var('VIRT_TEST'));
 
-    if (get_var("AUTOYAST")) {
-        loadtest "autoyast/installation";
-        loadtest "virt_autotest/reboot_and_wait_up_normal";
-    }
-    else {
-        load_inst_tests if check_var('VIRT_PART', 'install');
+    # Install hypervisor via autoyast or manually
+    unless (get_var('DISABLE_INSTALL')) {
+        if (check_var('HOST_INSTALL_AUTOYAST', 1)) {
+            # Install hypervisor via autoyast
+            loadtest "autoyast/prepare_profile";
+            loadtest "boot/boot_from_pxe";
+            loadtest "autoyast/installation";
+        } elsif (check_var('HOST_INSTALL_AUTOYAST', 0)) {
```

Comment:
Will this branch be executed if ```HOST_INSTALL_AUTOYAST``` is empty or not specified ?

Reply:
fixed. 
HOST_INSTALL_AUTOYAST=1: autoyast installation.
HOST_INSTALL_AUTOYAST=0 or null: manual installation.